### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by JamshedAlamQaderi** [Budibase/budibase#9740] [General]
Hi,
I want to make a website and CRM app with budibase. Is it possible to use budibase as a website with landing page and blogging feature and also a content management system?

**Comment by melohagan** [comment]
Hey @JamshedAlamQaderi 

Budibase is predominantly built for making internal Business tools as opposed to front-end customer portals and blogs, although technically it could be used for that purpose.

As a content management system, again Budibase is not really designed with this in mind but you could probably build something simple.
